src/applications/disability-benefits/526EZ/tests/config/privateMedicalRecordFacilities.unit.spec.jsx

```diff
@@ -0,0 +1,149 @@
+    schema,
+    uiSchema,
+    arrayPath,
+  } = formConfig.chapters.supportingEvidence.pages.privateMedicalRecordRelease;
+
+  it('should render 4142 form', () => {
+    const form = mount(
+      <DefinitionTester
+        arrayPath={arrayPath}
+        pagePerItemIndex={0}
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data={initialData}
+        uiSchema={uiSchema}
+      />,
+    );
+
+    expect(form);
+    expect(form.find('input').length).to.equal(8);
+    expect(form.find('select').length).to.equal(6);
+  });
+
+  it('should add a provider facility', () => {
+    const onSubmit = sinon.spy();
+    const form = mount(
+      <DefinitionTester
+        arrayPath={arrayPath}
+        pagePerItemIndex={0}
+        onSubmit={onSubmit}
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data={initialData}
+        formData={initialData}
+        uiSchema={uiSchema}
+      />,
+    );
+
+    //  All fields filled
+    fillData(
+      form,
+      'input#root_providerFacility_0_providerFacilityName',
+      'Local facility',
+    );
+    fillDate(
+      form,
+      'root_providerFacility_0_treatmentDateRange_from',
+      '1950-1-3',
+    );
+    fillDate(form, 'root_providerFacility_0_treatmentDateRange_to', '1951-1-3');
+    fillData(
+      form,
+      'select#root_providerFacility_0_providerFacilityAddress_country',
+      'USA',
+    );
+    fillData(
+      form,
+      'input#root_providerFacility_0_providerFacilityAddress_street',
+      '101 Street',
+    );
+    fillData(
+      form,
+      'select#root_providerFacility_0_providerFacilityAddress_state',
+      'AK',
+    );
+    fillData(
+      form,
+      'input#root_providerFacility_0_providerFacilityAddress_city',
+      'Anyville',
+    );
+    fillData(
+      form,
+      'input#root_providerFacility_0_providerFacilityAddress_postalCode',
+      '29414',
+    );
+
+    form.find('form').simulate('submit');
+    expect(onSubmit.called).to.be.true;
+    expect(form.find('.usa-input-error').length).to.equal(0);
+  });
+
+  it('should validate the providerFacilityName', () => {
+    const onSubmit = sinon.spy();
+    const form = mount(
+      <DefinitionTester
+        arrayPath={arrayPath}
+        pagePerItemIndex={0}
+        onSubmit={onSubmit}
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data={initialData}
+        formData={initialData}
+        uiSchema={uiSchema}
+      />,
+    );
+
+    // Fill the form with a name that's too long
+    fillData(
+      form,
+      'input#root_providerFacility_0_providerFacilityName',
+      'This input is entirely too long-winded to fit into this particular field--Whose idea was it to have this as a facility name anyhow',
+    );
+    form.find('form').simulate('submit');
+    expect(
+      form
+        .find('.usa-input-error-message')
+        .first()
+        .text(),
+    ).to.contain('100 characters');
```

Comment:
Unfortunately, this doesn't simulate what actually happens in the browser--at least not Chrome. We're not able to input more than 100 characters, so the validation doesn't get triggered.